snpe/tutorials/Inference_notebook_proposal.ipynb

```diff
@@ -0,0 +1,691 @@
+   "outputs": [
+    {
+     "data": {
+      "application/javascript": [
+       "\n",
+       "            setTimeout(function() {\n",
+       "                var nbb_cell_id = 2;\n",
+       "                var nbb_unformatted_code = \"ARTIFACT_PATH = Path(\\\"../../..\\\")\\n\\n# Estos path de abajo ven\\u00edan de antes, el de arriba es el que te has montado para que te furule a ti\\n# ARTIFACT_PATH = Path(\\\"../../../gcs_mount/artifacts/rating_spacing_simulator\\\")\\n# ARTIFACT_PATH = Path(\\\"/data/reputation-systems/snpe/artifacts/rating_spacing_simulator\\\")\";\n",
+       "                var nbb_formatted_code = \"ARTIFACT_PATH = Path(\\\"../../..\\\")\\n\\n# Estos path de abajo ven\\u00edan de antes, el de arriba es el que te has montado para que te furule a ti\\n# ARTIFACT_PATH = Path(\\\"../../../gcs_mount/artifacts/rating_spacing_simulator\\\")\\n# ARTIFACT_PATH = Path(\\\"/data/reputation-systems/snpe/artifacts/rating_spacing_simulator\\\")\";\n",
+       "                var nbb_cells = Jupyter.notebook.get_cells();\n",
+       "                for (var i = 0; i < nbb_cells.length; ++i) {\n",
+       "                    if (nbb_cells[i].input_prompt_number == nbb_cell_id) {\n",
+       "                        if (nbb_cells[i].get_text() == nbb_unformatted_code) {\n",
+       "                             nbb_cells[i].set_text(nbb_formatted_code);\n",
+       "                        }\n",
+       "                        break;\n",
+       "                    }\n",
+       "                }\n",
+       "            }, 500);\n",
+       "            "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Javascript object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ARTIFACT_PATH = Path(\"../../..\")\n",
+    "\n",
+    "# Estos path de abajo venían de antes, el de arriba es el que te has montado para que te furule a ti\n",
+    "# ARTIFACT_PATH = Path(\"../../../gcs_mount/artifacts/rating_spacing_simulator\")\n",
+    "# ARTIFACT_PATH = Path(\"/data/reputation-systems/snpe/artifacts/rating_spacing_simulator\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b7cefe6",
+   "metadata": {},
+   "source": [
+    "### B. Main function and its arguments"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1627882",
+   "metadata": {},
+   "source": [
+    "The function in the code cell below is designed to carry out inference on the basis of time series simulational output. Firstly a prior distribution of parameters is initialized containing the upper and lower bounds for the different parameters. Then the inferrer is instantiated (See inference_class.py) and the simulator is loaded alongside the inputs for the inference model. Lastly the inference model is trained, the posterior is built and the results are stored in the previously specified path as a .pkl file containing other relevant parameters alongside the posteriors.\n",
```

Comment:
`simulational` -> `simulation`
`Firstly a uniform prior distribution is initialized by defining the upper and lower bounds for the different parameters`

Reply:
Modified!